packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/ResourceType.java

```diff
@@ -6,7 +6,8 @@ public enum ResourceType {
 	INTERMEDIATES("intermediates"),
 	MODELS("models"),
 	PLANS("plans"),
-	PUBLICATIONS("publications");
+	PUBLICATIONS("publications"),
+	SIMULATION_RUNS("simulation_runs");
```

Comment:
Not entirely sure if this is the behaviour we want.
There seems to be a quirkiness with the way the enums are declared.

If we do
- /projects/:id/assets/plans/:id, this will errors out
if we do
- /projects/:id/aasets/PLANS/:id, this works

The route in the data-service is `plans` lowercased

The underlying exception is this
```
2022-11-25 11:31:17,523 DEBUG [org.jbo.res.res.i18n] (executor-thread-0) Unable to extract parameter from http request: javax.ws.rs.PathParam("resource_type") value is 'plans' for public javax.ws.rs.core.Response software.uncharted.terarium.hmiserver.resources.dataservice.ProjectResource.getAsset(java.lang.String,software.uncharted.terarium.hmiserver.models.dataservice.ResourceType,java.lang.String): java.lang.IllegalArgumentException: No enum constant software.uncharted.terarium.hmiserver.models.dataservice.ResourceType.plans
```

Reply:
this will probably be fixed in version 0.3.2 of the data-service. If we can get this in, then we can move up to the next version.

Reply:
actually, @dvince2 will look into this with his move up to 0.3.2. Bur we should get this PR in first